common/primitives/src/messages.rs

```diff
@@ -15,9 +15,10 @@ use utils::*;
 pub struct MessageResponse<AccountId, BlockNumber> {
 	#[cfg_attr(feature = "std", serde(with = "as_hex"))]
 	pub data: Vec<u8>, //  Serialized data in a user-defined schema format
-	pub signer: AccountId,       //  Signature of the signer
-	pub msa_id: MessageSenderId, //  Message source account id (the original sender)
-	pub index: u16,              // index in block to get total order
+	pub signer: AccountId,                     //  Signature of the signer
+	pub sender_msa_id: MessageSenderId,        //  Message source account id (the original sender)
```

Comment:
I dont know if we need this

Reply:
We should do a parking lot on it tomorrow, I think we might need it, if we want to keep a history of who delegated what for a delegator, like same schema can be published by one provider as well as others, having a placeholder of who sent it will help differentiate anyone look at it for example

Reply:
I think since when we revoke a delegation we still keep the history and when it got revoked, then it would be easy to track this back to the actual provider.

Reply:
So we agree @enddynayn  and @aramikm  we can remove that field and just rename it 


I will push an update